pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -600,6 +600,16 @@ public enum PGProperty {
       "0",
       "The timeout value in seconds max(2147484) used for socket read operations."),
 
+  /**
+   * Enable or disable mapping of PG types with TIMEZONE into SQL types with TIMEZONE.
+   * The default is {@code false}
+   */
+  SQL_TYPES_WITH_TIMEZONE(
+      "sqlTypesWithTimezone",
+      "false",
```

Comment:
We might consider `enum` style property instead as it might be easier to reason.

Reply:
@vlsi if we used `enum` style what would the two names be ?